### ISSUE TITLE
nf-core/tools=2.7.2 downloading scrnaseq: Cannot parse container string

### ISSUE THREAD

**Issue by mschubert** [nf-core/scrnaseq#193]
### Description of the bug

I'm trying to download `scrnaseq` via the `nf-core/tools` python package, but this produces a "Cannot parse container string" error.

As there were [similar](https://github.com/nf-core/sarek/issues/840) [issues](https://github.com/nf-core/sarek/issues/823) with [nf-core/sarek](https://github.com/nf-core/sarek) that were labeled as bugs, I assume this is the same case here.

### Command used and terminal output

```console
$ nf-core download scrnaseq
choose master
choose singularity
choose no compression


ERROR    Cannot parse container string in 'nf-core-scrnaseq-master/workflow/modules/local/mtx_to_seurat.nf':

             ${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
                     'docker://satijalab/seurat:4.1.0' :
                     'satijalab/seurat:4.1.0' }

         ⚠ Skipping this singularity image..
```

### System information

nf-core/tools version 2.7.2
nf-core/scrnaseq 2.1.0 and git master

**Comment by grst** [comment]
@FriederikeHanssen, may I ask what the problem was in the linked sarek issue (which is now fixed)?

**Comment by FriederikeHanssen** [comment]
@maxulysse  fixed it, so maybe he can give some more details. but iirc it was that for the container declaration for the ones that didn't come with biocontainers (and then didn't have a singularity image), we set it twice with the same string and then tools took issue with that. 

For deepvariant for example it looks like this now:
https://github.com/nf-core/sarek/blob/c87f4eb694a7183e4f99c70fca0f1d4e91750b33/modules/nf-core/deepvariant/main.nf#L10

**Comment by grst** [comment]
ok, that makes sense!

**Comment by FriederikeHanssen** [comment]
As a workaround, you can download the container manually and put it where the other containers live.

**Comment by grst** [comment]
Closed in #204
